apps/services/models.py

```diff
@@ -0,0 +1,192 @@
+from apps.common.db import Base
+
+from sqlalchemy import (MetaData, Column, Table, INTEGER, VARCHAR, FLOAT, ForeignKey, DATETIME, JSON, TIMESTAMP)
+from sqlalchemy.orm import relationship
+
+
+metadata_obj = MetaData()
+
+STATUS_MAPPING = {"Схвалено": 1, "Відхилено": 2, "Розглядається": 3, "Скасовано": 4}
+
+
+class Service(Base):
+    __tablename__ = 'service'
+
+    service_id = Column(INTEGER, primary_key=True, nullable=False)
+    service_name = Column(VARCHAR(length=255), nullable=False)
+
+    requisites = relationship("Requisites", back_populates="service")
+    user_request = relationship("UserRequest", back_populates="service")
+
+    def __repr__(self):
+        return f'{self.__class__.__name__}(service_id="{self.service_id}",service_name="{self.service_name}")'
+
+
+class UserRequest(Base):
+    __tablename__ = "user_request"
+
+    user_request_id = Column(INTEGER, primary_key=True, nullable=False)
+    date_created = Column(DATETIME, nullable=False)
+    comment = Column(VARCHAR(length=255))
+    user_id = Column(INTEGER, ForeignKey("user.user_id"), nullable=False)
+    service_id = Column(INTEGER, ForeignKey("service.service_id"), nullable=False)
+    faculty_id = Column(INTEGER, ForeignKey("faculty.faculty_id"), nullable=False)
+    university_id = Column(INTEGER, ForeignKey("university.university_id"), nullable=False)
+    status_id = Column(INTEGER, ForeignKey("status.status_id"), nullable=False)
+
+    users = relationship("User", back_populates="user_request")
```

Comment:
Rename users to `user` and fix back_populates accordingly, because of:
The UserRequest has only one User related to it.
The User can have multiple UserRequest entities.

P.S. It depends on the direction of the DB query needed...
```python
user = relationship("User", back_populates="user_requests")
```